models/renegadeXStats.go

```diff
@@ -58,14 +59,99 @@ type RenegadeXStats struct {
 
 func (statsData *RenegadeXStats) ParseJsonData(rawData map[string]interface{}) error {
 	fmt.Println("Attempting to parse raw data")
+
+	//Who won this game? integer represents teams
 	winner, err := strconv.Atoi(rawData["winner"].(string))
 	if err != nil {
 		return err
 	}
 	statsData.Winner = int16(winner)
+	//What was the win method? should be a string
+	winMethod := rawData["winMethod"].(string)
```

Comment:
You can check if the conversion was successful so that it won't panic when it cannot find the field etc. You can apply this to everywhere you used this structure. Example: 

```winMethod, ok := rawData["winMethod"].(string)
if !ok{
    return errors.New("winMethod is invalid")
}